lib/diagnostic.js

```diff
@@ -60,7 +60,7 @@ let deathStar = {
 // Question 4: Write the code that would set the `deathStar`'s `fire` method
 // in the context of `alderaan` but does not invoke the method,
 // so that we could call answer4() // Alderaan
-let answer4 = null;  // replace null with your response
+let answer4 = deathStar.fire.call(alderaan);  // replace null with your response
```

Comment:
`deathStar`'s `fire()` method needs to be bound to `alderaan` so that it's used as the context.  This is done by calling `Function`'s `bind()` on `fire()` with `alderaan` as the argument.

```javascript
deathStar.fire.bind(alderaan);
```